packit/upstream.py

```diff
@@ -452,7 +452,19 @@ def get_commit_messages(self, after: str = None, before: str = "HEAD") -> str:
             commits_range,
             "--",
         ]
-        return run_command(cmd, output=True, cwd=self.local_project.working_dir).strip()
+        try:
+            return run_command(
+                cmd, output=True, cwd=self.local_project.working_dir
+            ).strip()
+        except PackitCommandFailedError as ex:
```

Comment:
A little bit late to the party, but I'll just leave a note here: will this error always be raised due to the issue mentioned in the log lines bellow? How do we make sure, that the lines bellow also stay relevant in the future?

Reply:
We can move catching of the exception up the stack so it lives closer to the business logic. We just need to make sure we cover upstream and source-git.

To me, the current text sounds generic enough - it should still be better raising git errors which are hard to decipher without context for users.